README.md

```diff
@@ -2,13 +2,73 @@
 
 Accessibility Statement Frontend is a service for providing for accessibility statements across HMRC.
 
+## To add your service's accessibility statement
+
+Create a YAML file of the form:
+
+```yaml
+serviceKey: discounted-icecreams        # A unique name for the statement used for routing e.g. https://www.tax.service.gov.uk/accessibility-statement/discounted-icecreams
+serviceName: Discounted Icecreams       # The service name that will appear in the title of the accessibility statement
+serviceHeaderName: Icecreams            # The service name as it should appear in the grey Gov.UK header bar
+serviceDescription: |                   # A description of the service
+  Use this paragraph to describe your service. It can go over multiple lines
+  like this.
+serviceDomain: www.tax.service.gov.uk   # The domain name underwhich this service exists (exclude the https:// and the path)
+serviceUrl: /icecreams                  # The relative URL to the service (omitting www.tax.service.gov.uk)
+contactFrontendServiceId: icecreams    # The service id passed to contact-frontend and hmrc-deskpro
+complianceStatus: partial               # Whether this service is fully or partially compliant
```

Comment:
Maybe list the allowed values, i.e. Should be `fully|partial`

Reply:
Would it be worth including an example of a fully compliant statement?

Reply:
I don't think there needs to be a full example in the README, but I think where there is a list of allowed values, that should definitely be in the README.